tests/tests.wake

```diff
@@ -219,12 +218,11 @@ def runTest (testScript: Path): Result Unit Error =
         makeExecPlan ("./{testScript.getPathName.inTestDir}", wakeDir.inTestDir, Nil) (visibleFiles ++ wakeVisible)
         | setPlanDirectory testDirectory
         | setPlanLabel "testing: {testName}"
-        | setPlanLocalOnly True # On OS/X you cannot mount fuse within fuse
         | setPlanEnvironment ("HOME={testDirectory}",)
         | setPlanStdout logNever
         | setPlanStderr logNever
         | setPlanShare False
-        | runJob
+        | runJobWith localRunner # On OS/X you cannot mount fuse within fuse
```

Comment:
This might not even need the `localRunner` if we're committing to actively not supporting Mac.

Reply:
Sure, I'll leave that for someone else in a 'remove macos' PR

Reply:
"someone else"

...oh yeah....